compiler/src/iree/compiler/Dialect/Flow/Transforms/FormDispatchRegions.cpp

```diff
@@ -464,7 +461,7 @@ static bool isFusableWithConsumer(
 
   // Check if the iteration spaces of the producer and consumer are same.
   // TODO: This is unnecessary requirement, but needed to pass tests right now
-  if (!aggressiveFusion) {
+  if (!fuseMultiUse) {
```

Comment:
Just as an experiment... can you try removing lines `464`-`470` and see what happens. This is not really related to `fuseMultiUse`.

Reply:
Landed in #12589